### DISCUSSION TITLE
Predesigned Dashboard for certain roles

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#17496] [How to] [answered]
What would be the easiest way to design the same dashboard applied automatically for certain roles? For instance admin create users like 20 assigned to one role and he would like to set certain items so new user starts with ready to go dashboard (YF 6.4)

Cheers!

**Comment by rskrzypczak** [comment, marked as answer]
use the administration panel and create widgets for a specific role. they will then appear automatically to every user assigned to this role.
Note: for a widget to be added to the dashbourd by default, it must be set as mandatory.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/5c050539-f80d-4f51-b985-ed1f5dbaf05a)
